pkg/webapi/webapi.go

```diff
@@ -385,7 +386,7 @@ func (t WebAPI) upsertAccount(w http.ResponseWriter, r *http.Request, p httprout
 		return
 	}
 	o := giga.AccountCreateRequest{
-		PayoutThreshold: "0",
+		PayoutThreshold: decimal.RequireFromString("0"),
```

Comment:
There is a Zero value already, which is more efficient:
```
PayoutThreshold: decimal.Zero,
```

But actually, you don't need to initialise PayoutThreshold at all, so it can be:
```
o := giga.AccountCreateRequest{}
```